org_preferences.go

```diff
@@ -0,0 +1,55 @@
+package gapi
```

Comment:
Despite that team preferences methods are homed in `team.go`, I chose to home these org preferences methods in their own `org_preferences.go` file (rather than stick them in the existing `orgs.go` file) because...

* The Grafana API documentation documents the endpoints on a [dedicated Preferences page](https://grafana.com/docs/grafana/latest/developers/http_api/preferences/), separate from its [organization documentation](https://grafana.com/docs/grafana/latest/developers/http_api/org/).
* `grafana-api-golang-client` itself distinguishes between an `orgs.go` and `org_users.go` file, seemingly preferring multiple files with granular collections of orgs-related methods